### ISSUE TITLE
adblock not blocking ads

### ISSUE THREAD

**Issue by atrasatti** [wavebox/waveboxapp#1071]
* Wavebox Version: 4.10.5
* Operating System & Version: Mac OSX 10.14.5
* Account type (if applicable): LinkedIn

**Expected & actual behavior**
I enabled the native adblock plugin, but on LinkedIn all ads are visible. I expect most or all of them to be blocked and hidden.

**Steps to reproduce**
install ad-blocker, login to LinkedIn

**Is the bug persistent or intermittent?**
Persistent

**Comment by Thomas101** [comment]
Thanks, we've got some work underway to make ad-blocking much better :)

**Comment by atrasatti** [comment]
ublock origin?

**Comment by Thomas101** [comment]
> ublock origin?

Yup. We're working on late summer / early autumn for a release :)
